fluidBuild.config.cjs

```diff
@@ -65,8 +65,15 @@ module.exports = {
 			script: false,
 		},
 		"api-extractor:commonjs": ["tsc"],
-		"api-extractor:esnext": ["api-extractor:commonjs", "build:esnext"],
-		"build:rename-types": ["build:esnext", "api-extractor:esnext"],
+		"api-extractor:esnext": {
+			dependsOn: ["api-extractor:commonjs", "build:esnext", "build:rename-types"],
```

Comment:
Why does `:esnext` still depend on `api-extractor:commonjs`?

Reply:
Good catch; it shouldn't.

Reply:
I changed this, but per the discussion here https://github.com/microsoft/FluidFramework/pull/18903/files/d12a0a89ef985a42f3551974db7ee1dadf28c1e3#r1431792106 I added api-extractor:commonjs back to the dependencies. This will make fluid-build run :commonjs as a required dep if it's present.